src/data/MetaDataItem.py

```diff
@@ -105,7 +112,7 @@ def to_json(self) -> dict:
             'bb_fields': self.bb_fields.to_json(),
             'start_i': self.start_i,
             'end_i': self.end_i,
-            'reckless_intervals': self.reckless_intervals,
+            'reckless_intervals': [{"start": ri[0], "end": ri[1]} for ri in self.reckless_intervals],
```

Comment:
limit of firestore is it doesnt allow nested arrays (tuples count as arrays) so have to convert in to_json and from_json